### DISCUSSION TITLE
How to apply splash layout to my git page

### DISCUSSION THREAD

**Discussion by ROBOTIS-David** [mmistakes/minimal-mistakes#2672] [Q&A] [answered]
Hi, I would like to start my words "Thank you for sharing the theme".  I really love your theme. 

I want to change a layout of [the Main page of our e-Manual](https://emanual.robotis.com/) like the [Main page of the minimal mistake](https://mmistakes.github.io/minimal-mistakes/). But I've kept failed to change the layout (splash). 

When I modified your [home.md](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_pages/home.md) file by cloning your repository ([minimal-mistakes](https://github.com/mmistakes/minimal-mistakes)), the layout works perfectly. (See 
![the_Right_format](https://user-images.githubusercontent.com/48426280/91549514-02b98180-e962-11ea-963d-06114278448f.png) )

But when I do the same thing with my [home.md](https://github.com/ROBOTIS-GIT/e-Manual_Test/blob/test/home.md) that I created for a test, the layout is not properly applied to my file as the following image. 
![the_wrong_format](https://user-images.githubusercontent.com/48426280/91549828-870c0480-e962-11ea-9891-92e39973e844.png)

I checked **my splash.html** and **page__hero.html**, I do not see problem so far. 

Could you give me little assistance or guide? 

For your reference, 

1. Here is the test repository that I tested for applying the splash layout. 

- [https://github.com/ROBOTIS-GIT/e-Manual_Test](https://github.com/ROBOTIS-GIT/e-Manual_Test)

2. I did bundle update 

3. Jekll version : jekyll (3.6.3)

4. Bundle version: Bundler version 2.1.2

5. OS: Ubuntu 16.04 

If you need any further information regarding this, please let me know.

Thank you so much for reading.

**Comment by mmistakes** [comment, marked as answer]
1. You're not using the latest version of the theme... you're using a 3 year old version. I would suggest [updating](https://mmistakes.github.io/minimal-mistakes/docs/upgrading/).
2. You're loading custom CSS that is overriding the theme's so the styling of MM is not loading.

I'm not at all familiar with the CSS you are loading instead so can't really help. Use the theme in it's default state and go from there before trying to add CSS on top of it.

**Comment by ROBOTIS-David** [comment]
Thank you for your comment !!!
